tests/python/pants_test/engine/legacy/test_graph_integration.py

```diff
@@ -0,0 +1,178 @@
+      Computing Task(hydrate_target, testprojects/src/java/org/pantsbuild/testproject/bundle:missing-bundle-fileset, =HydratedTarget)
+        Computing Task(hydrate_bundles, BundlesField(address=BuildFileAddress(testprojects/src/java/org/pantsbuild/testproject/bundle/BUILD, missing-bundle-fileset), bundles=[BundleAdaptor(fileset=[\'a/b/file1.txt\']), BundleAdaptor(fileset=<pants.engine.legacy.structs.RGlobs object at 0x7f889e374c50>), BundleAdaptor(fileset=<pants.engine.legacy.structs.Globs object at 0x7f889e374bd0>), BundleAdaptor(fileset=<pants.engine.legacy.structs.ZGlobs object at 0x7f889e374b50>), BundleAdaptor(fileset=[\'file1.aaaa\', \'file2.aaaa\'])], filespecs_list=[{u\'exclude\': [], u\'globs\': [u\'a/b/file1.txt\']}, {u\'exclude\': [], u\'globs\': [u\'**/*.aaaa\', u\'**/*.bbbb\']}, {u\'exclude\': [], u\'globs\': [u\'*.aaaa\']}, {u\'exclude\': [], u\'globs\': [u\'**/*.abab\']}, {u\'exclude\': [], u\'globs\': [u\'file1.aaaa\', u\'file2.aaaa\']}], path_globs_list=[PathGlobs(include=(u\'testprojects/src/java/org/pantsbuild/testproject/bundle/a/b/file1.txt\',), exclude=(), glob_match_error_behavior=GlobMatchErrorBehavior(failure_behavior=u\'ignore\')), PathGlobs(include=(u\'testprojects/src/java/org/pantsbuild/testproject/bundle/**/*.aaaa\', u\'testprojects/src/java/org/pantsbuild/testproject/bundle/**/*.bbbb\'), exclude=(), glob_match_error_behavior=GlobMatchErrorBehavior(failure_behavior=u\'ignore\')), PathGlobs(include=(u\'testprojects/src/java/org/pantsbuild/testproject/bundle/*.aaaa\',), exclude=(), glob_match_error_behavior=GlobMatchErrorBehavior(failure_behavior=u\'ignore\')), PathGlobs(include=(u\'testprojects/src/java/org/pantsbuild/testproject/bundle/**/*.abab\',), exclude=(), glob_match_error_behavior=GlobMatchErrorBehavior(failure_behavior=u\'ignore\')), PathGlobs(include=(u\'testprojects/src/java/org/pantsbuild/testproject/bundle/file1.aaaa\', u\'testprojects/src/java/org/pantsbuild/testproject/bundle/file2.aaaa\'), exclude=(), glob_match_error_behavior=GlobMatchErrorBehavior(failure_behavior=u\'ignore\'))]), =HydratedField)
+          Computing Snapshot(Key(val="PathGlobs(include=(u\\\'testprojects/src/java/org/pantsbuild/testproject/bundle/*.aaaa\\\',), exclude=(), glob_match_error_behavior<=GlobMatchErrorBehavior>=GlobMatchErrorBehavior(failure_behavior=error))"))
+            Throw(PathGlobs expansion failed: Throw(Globs did not match. Excludes were: []. Unmatched globs were: ["testprojects/src/java/org/pantsbuild/testproject/bundle/*.aaaa"]., "<pants native internals>"))
+              Traceback (no traceback):
+                <pants native internals>
+              Exception: PathGlobs expansion failed: Throw(Globs did not match. Excludes were: []. Unmatched globs were: ["testprojects/src/java/org/pantsbuild/testproject/bundle/*.aaaa"]., "<pants native internals>")
+"""
+    self.assertIn(expected_msg, pants_run.stderr_data)
+
+  def test_exception_invalid_option_value(self):
```

Comment:
This effectively just tests options parsing... would remove it (or if you don't see a test of the `choices` impl, move something like this test to the relevant test class).

Reply:
I have removed this test. Thanks!